mypy/pyinfo.py

```diff
@@ -31,9 +32,7 @@ def getsitepackages() -> list[str]:
         if hasattr(site, "getusersitepackages") and site.ENABLE_USER_SITE:
             res.insert(0, site.getusersitepackages())
     else:
-        from distutils.sysconfig import get_python_lib
-
-        res = [get_python_lib()]
+        res = [sysconfig.get_paths()["purelib"]]
```

Comment:
Something I don't recall, are purelib and sitepackages 1-to-1? I also don't recall if platdir is included in getsitepackages.

Reply:
Ah yeah, I think what I have is equivalent to `get_python_lib`, but I think you're right that platlibdir is in getsitepackages